west/tools.py

```diff
@@ -181,8 +181,8 @@ def monitor(self, module_path, args):
         monitor_path = Path(module_path, "tools/idf_monitor/idf_monitor.py")
         cmd_path = Path(os.getcwd())
         if platform.system() == 'Windows':
-            cmd_exec(("python.exe", monitor_path, "-p", esp_port,
-                     "-b", args.baud, elf_path, "--eol", args.eol), cwd=cmd_path)
+            cmd_exec((sys.executable, str(monitor_path), "-p", esp_port,
```

Comment:
It almost looks like the `platform.system()` condition could be removed, right? Can you make the `cmd_exec` call universal for all platforms?

Reply:
It might work, but I don't have a non-Windows host to try that change out with.

Reply:
@ajf58 from what I see here, the only differences between Windows and non-Windows cases are the stringification of 2 arguments. Therefore I asked if you remove the condition. I'll test it here and let you know.

Reply:
OK, that's done. Please let me know how your testing gets on.